src/quacc/recipes/emt/md.py

```diff
@@ -0,0 +1,89 @@
+"""
+Molecular Dynamics recipes for EMT.
+"""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from ase.calculators.emt import EMT
+from ase.md.verlet import VelocityVerlet
+
+from quacc import job
+from quacc.runners.ase import Runner
+from quacc.schemas.ase import summarize_md_run
+from quacc.utils.dicts import recursive_dict_merge
+
+if TYPE_CHECKING:
+    from typing import Any
+
+    from ase.atoms import Atoms
+    from ase.md.md import MolecularDynamics
+
+    from quacc.runners.ase import MDParams
+    from quacc.schemas._aliases.ase import DynSchema
+    from quacc.utils.files import Filenames, SourceDirectory
+
+
+@job
+def md_job(
+    atoms: Atoms,
+    dynamics: MolecularDynamics = VelocityVerlet,
+    dynamics_kwargs: dict[str, Any] | None = None,
+    initial_temperature: float | None = None,
+    md_params: MDParams | None = None,
+    copy_files: SourceDirectory | dict[SourceDirectory, Filenames] | None = None,
+    **calc_kwargs,
+) -> DynSchema:
```

Comment:
I have included some sensible default keyword arguments here to make it more accessible to users. What we include/not include is subject to discussion, of course.